labs/litellm_service/local.py

```diff
@@ -0,0 +1,28 @@
+from ollama import Client
+
+
+class RequestLocalLLM:
+    def __init__(self):
+        self.client = Client(host="http://localhost:11434")
+
+    def format_response(self, response):
+        return {"choices": [response]}
+
+    def completion_without_proxy(self, messages, model=None):
+        """
+        messages expected to be in the following format:
+        [
+            {
+                "role": "user",
+                "content": ""
+            }
+        ]
+        Where role can be "user", "assistant", "system".
+        And content is the body of the message.
+        """
+        model = "llama3.2"
```

Comment:
Any reason why this is a constant?

Reply:
because it's also returned at the end, and for now we're just going to use llama3.2
later on we can probably move it to a function argument or an env variable